api/app/Http/Controllers/Api/VotesController.php

```diff
@@ -20,13 +21,29 @@ public function store(VoteModel $request)
         $currentVote = Vote::where(['observation_id' => $request->observation_id, 'user_id' => auth()->user()->id])->first();
 
         if (! is_null($currentVote)) {
-            return response()->json('You has already voted');
+            return response()->json('You have already voted');
         }
 
         $vote = new Vote($request->all());
         $vote->user_id = auth()->user()->id;
         $vote->save();
 
+        $this->checkObservationThreshold(Observation::find($request->observation_id));
+
         return $vote;
     }
+
+    private function checkObservationThreshold(Observation $observation)
```

Comment:
E.g. all the observations that are 'bad' will always be shown. 

E.g. 10 users downvote the observation.
If an 11 user register, it will still show this observation?